journey/pepr-build-wasm.ts

```diff
@@ -9,17 +9,24 @@ import { resolve } from "path";
 
 import { cwd } from "./entrypoint.test";
 
+// test npx pepr build -o dst
+const outputDir = "dst";
```

Comment:
Have we tested that this works with deeper / multi-part paths -- something like `this/folder/might/not/exist/yet/dst`?

Reply:
should work now, good eye